front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -759,13 +772,13 @@ class SettingsCustomizeTab extends PureComponent<
             </WideSectionField>
           </Section>
 
-          {tenant.data.attributes.settings?.events_page?.allowed && (
-            <Section>
-              <SectionTitle>
+          <FeatureFlag name="events_page" onlyCheckAllowed>
+            <EventsSection>
```

Comment:
The margins were a bit too big. Admittedly, it's hard with our incoherent margin in the admin. The least bad solution I could come up with is to have custom margins for everything events.